Dockerfile

```diff
@@ -47,8 +48,9 @@ RUN echo "server { \n \
 	} \n \
 }\n " > /etc/nginx/sites-available/default
 
-
-RUN ln -s /githubscripts/gh-hook-bugzilla.py  /usr/share/nginx/html/gh-hook-bugzilla.py
-RUN ln -s /githubscripts/gh-hook-mr.py  /usr/share/nginx/html/gh-hook-mr.py
+RUN ln -s /githubscripts/gh-hook-bugzilla.py  /var/www/html/gh-hook-bugzilla.py
+RUN ln -s /githubscripts/gh-hook-mr.py  /var/www/html/gh-hook-mr.py
```

Comment:
I think here might be reasonable to put to www all directory. In case if I will need to add additional scripts.  Then I will just commit them to git.

Reply:
We could just clone into the /var/www/html (or /srv) directory directly then don't need to symlink

Reply:
yes. that also has to work. But we need to keep updates by cron.